source-postgres/backfill.go

```diff
@@ -203,6 +203,9 @@ func (db *postgresDatabase) keylessScanQuery(_ *sqlcapture.DiscoveryInfo, schema
 	if db.config.Advanced.MinimumBackfillXID != "" {
 		fmt.Fprintf(query, ` AND (((xmin::text::bigint - %s::bigint)<<32)>>32) > 0 AND xmin::text::bigint >= 3`, db.config.Advanced.MinimumBackfillXID)
 	}
+	if db.config.Advanced.MaximumBackfillXID != "" {
+		fmt.Fprintf(query, ` AND (((%s::bigint - xmin::text::bigint)<<32)>>32) > 0 AND xmin::text::bigint >= 3`, db.config.Advanced.MaximumBackfillXID)
```

Comment:
There were three obvious ways to negate the "minimum XID" filter expression when adding the maximum, and I arbitrarily went with "swap the operands of the subtraction". I don't see any possible way it could matter, but this could instead have been done by changing `> 0` to `< 0` or by slapping a `NOT` on it.

Let me know if you think one of those alternatives would be clearer.